dkist/dataset/tests/test_crop_dataset.py

```diff
@@ -1,9 +1,14 @@
+import pytest
+
 import astropy.units as u
 from astropy.coordinates import SkyCoord, SpectralCoord, StokesCoord
 from astropy.time import Time
 
+gwcs = pytest.importorskip("gwcs", "0.22.2")
```

Comment:
I was thinking more in the test which needed this, but if they all do this is fine.